### ISSUE TITLE
How can I test a model with the Cross Entropy evaluation method?

### ISSUE THREAD

**Issue by AntonioVispi** [open-mmlab/mmclassification#1075]
### Prerequisite

- [X] I have searched related issues but cannot get the expected help.
- [X] I have read [the documentation](https://mmclassification.readthedocs.io/) and don't know what to do.

### Describe the question

Greetings,
I am trying to test a trained model; for the other metrics such as accuracy, recall, precision and f1-score I have no problems. However, I was wondering if Cross Entropy metric had been implemented as an additional means of evaluating the performance of the classifier starting from `class_scores`. Mine is a multiclass problem. 
Thanks for your attention

**Comment by mzr1996** [comment]
Cross-entropy is not a typical test metric and we don't support it during tests.
If you want to modify the available test metric, you can modify the `evaluate` method of the `BaseDataset` in [this file](https://github.com/open-mmlab/mmclassification/blob/master/mmcls/datasets/base_dataset.py#L126).

**Comment by AntonioVispi** [comment]
It's clear, thank you very much.
